<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingwidget</class>
 <widget class="QWidget" name="settingwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centerWidget{
    border: none;
    border-radius: 20px;
    background-color: rgb(207,231,244);
}
QPushButton#closeBtn
{
    border: none;
    image:url(://resource/button/shut.png);
    min-width: 32px;
    min-height: 32px;
    border-radius: 16px;
}
QPushButton#closeBtn:hover
{
    background-color: rgb(231,37,54);
    border-radius: 16px;
}
QWidget#btnWidget QPushButton
{
    border:none;
    border-radius: 5px;
    background-color: rgb(6,163,220);
    min-width: 80px;
    min-height: 30px;
    color: white;
}
QWidget#btnWidget QPushButton:hover
{
    background-color: rgb(16,196,245);
}
QStackedWidget
{
    background-color: qlineargradient(spread:pad,x1:0, y1:0, x2:0, y2:1, stop:0 rgba(232, 241, 249, 0), stop:0.1 rgba(232, 241, 249, 1), stop:0.2 rgba(232, 241, 249, 255), stop:0.7 rgba(232, 241, 249, 255),stop:1 rgba(252, 252, 252, 255));
    border-radius:5px;
}
QGroupBox
{
    border: none;
}</string>
  </property>
  <widget class="QWidget" name="centerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>451</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QCheckBox" name="checkBox1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>人物①</string>
      </property>
     </widget>
     <widget class="QLabel" name="model_show_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>241</width>
        <height>321</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="G:/QT/15 项目之桌面壁纸/Wallpaper/resourse.qrc">:/resource/desktopRole/summerGril/0.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="model_show_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>251</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="G:/QT/15 项目之桌面壁纸/Wallpaper/resourse.qrc">:/resource/desktopRole/littleBoy/0.png</pixmap>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>人物②</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>人物心情</string>
      </property>
      <widget class="QRadioButton" name="sad">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>63</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>哀怨（伤）</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="shy">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>117</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>害羞(羞)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="happy">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>36</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>开心（喜）</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="pi">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>90</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>调皮（皮）</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>模型③</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="model_show_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>291</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="pixmap">
       <pixmap resource="G:/QT/15 项目之桌面壁纸/Wallpaper/resourse.qrc">:/resource/desktopRole/blackGril/action1-happy/0.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>model_show_3</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>自定义桌面壁纸</string>
      </property>
      <widget class="QPushButton" name="selectBtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>选择壁纸</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>281</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>程序设置</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="btnWidget" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>101</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="wall_setting">
       <property name="text">
        <string>壁纸设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="model_1">
       <property name="text">
        <string>人物①</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="model_2">
       <property name="text">
        <string>人物②</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="model_3">
       <property name="text">
        <string>人物③</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
